src/resources/views/auth/fullpage_redirect.blade.php

```diff
@@ -6,6 +6,8 @@
 
         <title>Redirecting...</title>
 
+        <script src="https://unpkg.com/@shopify/app-bridge{{ \Osiset\ShopifyApp\Util::getShopifyConfig('appbridge_version') ? '@'.config('shopify-app.appbridge_version') : '' }}"></script>
```

Comment:
can you move this logic to the controller?

Reply:
My PHP/Laravel is a bit rusty but I'll do my best haha